content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,339 @@
+
+Follow [the instructions listed here][33] to set up Log Collection.
+
+<div class="alert alert-warning"><b>Note</b>: The <b>Dataflow API</b> must be enabled to use Google Cloud Dataflow. See <a href="https://cloud.google.com/apis/docs/getting-started#enabling_apis"><b>Enabling APIs</b></a> in the Google Cloud documentation for more information.</div>
+
+Alternatively, to collect logs from applications running in GCE or GKE, you can also use the [Datadog Agent][32].
+
+## Google Cloud integrations
+
+Our [Integrations page][26] provides a full listing of the available sub-integrations for Google Cloud. 
+Many of these integrations are installed by default when Datadog recognizes data being ingested in from your Google Cloud account.
+
+## Leveraging the Datadog Agent for greater visibility
+
+After the Google Cloud integration is configured, Datadog automatically starts collecting Google Cloud metrics. However, you can leverage the Datadog Agent to gather deeper insights into your infrastructure.
+
+The [Datadog Agent][25] provides the [most granular, low-latency metrics from your infrastructure][47], delivering real-time insights into CPU, memory, disk usage, and more for your Google Cloud hosts. 
+The Agent can be installed on any host, including GKE - [Google Kubernetes Engine, Agent][28].
+
+The Agent also supports [a wide range of integrations][29], enabling you to extend visibility into specific services and databases running on your hosts. 
+
+[Traces collected through the Trace Agent][31] enable comprehensive Application Performance Monitoring (APM), helping you understand end-to-end service performance. 
+
+For a full list of benefits with installing the Agent on your cloud instances, see [Why should I install the Datadog Agent on my cloud instances?][27]
+
+
+
+## Explore related services
+
+### Private Service Connect 
+**(Only available in US5 & US1)**
+
+[Private Service Connect (PSC)][45] is a Google Cloud networking product that enables you to access [Google Cloud services][43], [third-party partner services][44], and company-owned applications directly from your Virtual Private Cloud (VPC).
+
+Get started with the [Google Cloud Private Service Connect guide.][46]
+
+### Google Cloud Run
+
+To begin monitoring serverless functions running on Google Cloud Platform, enable the [Google Cloud Platform][6] integration. 
+
+The [Google Cloud Run][34] page provides a list of out-of-the-box metrics collected by the Google Cloud integration, along with instructions for [Log Collection through Dataflow][35].
+
+### Cloud Cost Management (CCM)
+
+[Cloud Cost Management][50] provides insights for engineering and finance teams to understand how infrastructure changes impact costs, allocate spend across your organization, and identify potential improvements.
+
+To use Google Cloud Cost Management in Datadog, [follow these steps][51].
+
+### Security
+
+{{< tabs >}}
+{{% tab "Cloud SIEM" %}}
```

Comment:
What's your reason for using tabs for the two security products?

Reply:
Tbh I just thought it was cute & it made the page a little shorter - I've switched it back now though